_posts/2015-06-26-ux-part-3.md

```diff
@@ -0,0 +1,122 @@
+
+*Different toolbar arrangements visulaised.*
+
+But the main point is that it takes mere seconds to change the layout in the mock-up so we can see what it looks like, as against the considerable mount of time and effort it takes for a developer to recode the UI, rebuild and show it to all of the team. This way the valuable developer time is not needlessly taken up with throw-away prototypes.
+
+And here is the same process in OMERO.web, demonstrating the close alignment of this part of the UI in the two clients at this stage.
+
+<img src="/images/state-7.png" alt="Screenshots of mock-ups showing various toolbar placement options in OMERO.web." style="width: 600px;"/>
+
+*The toolbar arrangements in the OMERO.web UI.*
+
+Then, when the team has discussed all the options and decided on the way forward, the developers proceed with the coding, using the agreed mock-ups as the specification for layout of the controls and functionality.
+
+So we now had the rest of what we had scoped out ready for release with version 5.1.0.
+
+<img src="/images/state-8.png" alt="Screenshot showing the implemntation for the 5.1.0 release." style="width: 600px;"/>
+
+*OMERO.insight and OMERO.web UIs for the 5.1.0 release.*
+
+But when we showed it to some users for feedback, they looked at it and said “all very nice”, but don’t forget we want to be able to save multiple rendering settings for each user for each image. So ...
+
+##Third Round
+
+... back to the drawing board.
+
+<img src="/images/state-9.png" alt="Mock-ups of multiple rendering settings per user per image." style="width: 600px;"/>
+
+*Multiple rendering settings per user.*
+
+Yes, but no. They liked that, but wanted to be able to give each setting a unique name.
+
+So we needed to add editing buttons and dialogue boxes.
+
+<img src="/images/state-10.png" alt="Mock-ups of multiple rendering settings per user per image with name and editing dialogs." style="width: 600px;"/>
+
+*Let them have names!*
+
+And this will hopefully be implemented in the next round of UI improvements.
+
+Another important advantage with this workflow is that with all the scoping done well in advance of actual implementation, it helps to avoid conflict with other changes in the pipeline. So now if another change is muted that affects the Preview pane, Full Viewer or rendering settings, it is relatively easy to make sure that it fits in with these changes.
+
+##Release
+
+The UX team’s job doesn’t end with code implementation, as we don’t just abandon the user at this stage. With the OME’s continuous integration development system, the code, the developer documentation and the User Help are all updated and held in repositories ready for release.
+
+So when all the testing is complete and everyone’s happy, Sébastien presses The Big Red Button and the version release of all the elements is done at once. 
```

Comment:
This slightly makes Seb sound like the project gatekeeper! Maybe better to just say 'The Big Red Button is pressed'